calico/src/main/scala/calico/html/Modifier.scala

```diff
@@ -61,12 +75,12 @@ object Modifier:
       }
     }
 
-private trait Modifiers[F[_]](using F: Async[F]):
-  inline given forUnit[E]: Modifier[F, E, Unit] =
-    _forUnit.asInstanceOf[Modifier[F, E, Unit]]
+private trait ModifierLowPriority:
+  given forFoldable[F[_], E <: fs2.dom.Node[F], G[_]: Foldable, A](
+      using M: Modifier[F, E, A]): Modifier[F, E, G[A]] =
+    (ga, e) => ga.foldMapM(M.modify(_, e)).void
```

Comment:
Unfortunately this is causing a lot of angst, because `Tuple` also has foldable instances. Sadly I wonder if I should remove it in favor of instances specifically for `List` and stuff.

Indeed, this is what caused the original bug when I attempted to implement this.

Reply:
Replaced with a `List`-specific instance.

Reply:
... and `Option` instance too :) hopefully those are the main ones.